movie/src/components/body/Body.css

```diff
@@ -0,0 +1,4 @@
+.body {
```

Comment:
Please change this name base on which name you are going to pick from this comment.https://github.com/phandyph/popularmovie/pull/2/files?diff=split&w=0#r1138178626